FooCamp/atalaya
# Header/Navigation component

- Adding the html structure needed for the modal and the icons used for the header in the header.hbs template file
- Adding the styles for the header component for mobile and desktop viewports, and the styles for the menu modal in the header.scss file
- Adding the header.js with the functionality for the menu modal on mobile

**How to test**
Go to the Homepage and see the component at top of the page

https://user-images.githubusercontent.com/28935600/197276134-5bd21954-4494-4fce-ac90-b91bb0200d1d.mov
